Google.PowerShell/ReleaseFiles/GoogleCloud.psd1

```diff
@@ -92,20 +92,29 @@ PrivateData = @{
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        # Tags = @()
+        Tags = @('Google', 'Cloud')
 
         # A URL to the license for this module.
-        # LicenseUri = ''
+        LicenseUri = 'https://github.com/GoogleCloudPlatform/google-cloud-powershell/blob/master/LICENSE'
 
         # A URL to the main website for this project.
-        # ProjectUri = ''
+        ProjectUri = 'https://github.com/GoogleCloudPlatform/google-cloud-powershell'
 
         # A URL to an icon representing this module.
         # IconUri = ''
 
         # ReleaseNotes of this module
-        # ReleaseNotes = ''
+        ReleaseNotes = @"
+    This module contains a set of cmdlets that lets you manage Google Cloud Platform resources.
+    Currently, the supported resources are Google Cloud Storage resources, Google Compute Engine

```

Comment:
I think you should drop the "resources" after each item.  It seems a little redundant.